packages/widgets/src/menubar.ts

```diff
@@ -727,6 +737,7 @@ export class MenuBar extends Widget {
   }
 
   private _activeIndex = -1;
+  private _focusIndex = 0;
```

Comment:
I'm a little worried about adding another state variable to this component. Why do we need to allow the active index to diverge from the focus index?

Reply:
After discussing this over a call, I better understand the need for another state variable, but could you leave some comments explaining what these variables mean and why we need both of them?